analysis/org.eclipse.tracecompass.analysis.counters.core/src/org/eclipse/tracecompass/analysis/counters/core/aspects/ITmfCounterAspect.java

```diff
@@ -52,4 +53,25 @@ default boolean isHiddenByDefault() {
     default boolean isCumulative() {
         return true;
     }
+
+    /**
+     * Get the groups
+     *
+     * @return the groups
+     * @since 2.2
+     */
+    default Class<? extends ITmfEventAspect<?>>[] getGroups(){
+        return new Class[0];
+    }
+
+    /**
+     * Gets the type of this counter
+     *
+     * @return the type of this counter
+     * @since 2.2
```

Comment:
API change requires bump to 2.3.0 in MANIFEST.MF and since tag changed to 2.3 here.

Reply:
2.3

Reply:
also missing a @return tag